CONTRIBUTING.md

```diff
@@ -17,6 +18,31 @@ This document is a living summary of conventions and best practices for developm
   - [Documentation Guidelines](#documentation-guidelines)
   - [More Information](#more-information)
 
+## SIGN ALL OF YOUR COMMITS
+
+Whenever you make a commit, it is required to be signed. If you do not, you will have to re-write the git history to get all commits signed before they can be merged, which can be quite a pain.
+Use the "-s" or "--signoff" flags to sign a commit.
+Example calls:
```

Comment:
The way text flow works in Markdown can be kind of odd. Some text is showing up all on one line that looks like it was intended to be split across multiple lines:
![image](https://user-images.githubusercontent.com/22344007/115064119-8ce73380-9eba-11eb-8b48-53f10f6ebd96.png)

To improve the text flow, it might help to add bullet points in some places, or insert more than one line break (example below). To see how the Markdown looks, you can preview it in VS Code before pushing 🙂 

```markdown
Whenever you make a commit, it is required to be signed. If you do not, you will have to re-write the git history to get all commits signed before they can be merged, which can be quite a pain.

Use the "-s" or "--signoff" flags to sign a commit.

Example calls:
* `git commit -s -m "Adding a test file to new_branch"`
* `git commit --signoff -m "Adding a test file to new_branch"`
```

Reply:
@t1m0thyj missed this comment at first, working on it now.